deploy-board/deploy_board/webapp/cluster_view.py

```diff
@@ -311,6 +311,23 @@ def get_base_images(request):
     })
 
 
+def add_acceptance(request, base_images):
+    fetched_names = set()
+    name_acceptance_map = {}
+    for img in base_images:
+        name = img['abstract_name']
```

Comment:
is abstract_name unique? I think it's user entered field and does not have uniqueness contraint. In that case, can we populate status based on unique key ex: provider_name?

Reply:
`get_acceptance_by_name` takes abstract_name and returns all images with that abstrac_name. That's why I used the dict `name_acceptance_map` to store the mapping from provider_name to acceptance.